docs/developers/stored-requests.md

```diff
@@ -0,0 +1,138 @@
+# Stored Requests
+
+This document gives a technical overview of the Stored Impressions feature.
+
+Docs outlining the motivation and uses will be added sometime in the future.
+
+## Quickstart
+
+Configure your server to read stored requests from the filesystem:
+
+```yaml
+ortb2_config:
+  filesystem: true
+```
+
+Choose an ID to reference your request. Throughout this doc, replace {id} with the ID you've chosen.
+
+Add the file `openrtb2_configs/for_requests/{id}.json` and populate it with some Impression data.
+
+```json
+{
+  "id": "test-imp-id",
+  "banner": {
+    "format": [
+      {
+        "w": 300,
+        "h": 250
+      },
+      {
+        "w": 300,
+        "h": 600
+      }
+    ]
+  },
+  "ext": {
+    "appnexus": {
+      "placementId": 10433394
+    }
+  }
+}
+```
+
+Start your server.
+
+```bash
+go build .
+./prebid-server
+```
+
+And then `POST` to [`/openrtb2/auction`](../endpoints/openrtb2/auction.md) with your chosen ID.
+
+```json
+{
+  "id": "test-request-id",
+  "imp": [
+    {
+      "ext": {
+        "prebid": {
+          "managedconfig": {
+            "id": "{id}"
+          }
+        }
+      }
+    }
+  ]
+}
+```
+
+The Auction will occur as if the Request had used the content from `openrtb2_configs/for_requests/{id}.json` instead.
+
+## Partially Stored Requests
+
+You can also store _part_ of the Impression on the server. For example:
+
+```json
+{
+  "banner": {
+    "format": [
+      {
+        "w": 300,
+        "h": 250
+      },
+      {
+        "w": 300,
+        "h": 600
+      }
+    ]
+  },
+  "ext": {
+    "appnexus": {
+      "placementId": 10433394
+    }
+  }
+}
+```
+
+Note that OpenRTB requires each `imp` to have an `id` property.
+
+For a given HTTP Request to be valid, it must contain these missing properties:
+
+```json
+{
+    "id": "test-request-id",
+    "imp": [
+      {
+        "id": "test-imp-id",
+        "ext": {
+          "prebid": {
+            "managedconfig": {
+              "id": "{id}"
+            }
+          }
+        }
+      }
+    ]
+  }
+```
+
+If the Stored Request and the HTTP Request have conflicting properties,
```

Comment:
'Stored Request' appears to be a proper noun (the name of this feature), but I think all uses of 'HTTP Request' can use a lower-case 'r' (unless it's referring to some type defined in the Go code?)